### DISCUSSION TITLE
How to use a proxy?

### DISCUSSION THREAD

**Discussion by muffl0n** [stepci/stepci#124] [Q&A]
Is there any way to use a proxy? Stepci does not follow environment variables like `http_proxy` or `https_proxy` ootb and I didn't find anything in the code.
Am I missing the obvious?

Thank you very much!

**Comment by mishushakov** [comment]
No proxy support at the moment. Maybe you'd like to contribute?

https://github.com/sindresorhus/got/blob/main/documentation/tips.md#proxying

**Reply by muffl0n** [reply]
I'd love to! But my node experience does not exceed `npm install`, so I'd not expect some useful output. 😄

**Reply by mishushakov** [reply]
No problem, I'll keep you updated once we have some solution

**Reply by muffl0n** [reply]
I gave it a try anyway and came up with this simple, yet powerful solution: https://github.com/stepci/stepci/pull/126
It may not be the best solution though.

**Comment by mishushakov** [comment]
Hey @muffl0n, we have finally added proxy support. See:

https://docs.stepci.com/guides/using-proxy.html
